### PR TITLE
Infrastructure Updates

### PR DESCRIPTION
**Pull request by vmstan**
Documenting changes from this last week.

### CODE REVIEW

**Comment by vmstan** [comment]
FYI @djwfyi docs site is no longer generated by Netlify to reduce cost so pull request previews are not available. I may re-create the staging site but for now reviews have to be done the old fashioned way.

File: docus/content/infrastructure/relays.md [hunk 1]
```diff
@@ -12,3 +12,28 @@ vmst.io does not currently run a Fediverse relay, but we participate in the foll
 
 Relays allow all content from participating instances to be directly federated to everyone else, without waiting for a direct user connection to be established.
 This helps with content discovery and more to complete conversation threads and hashtags for our users.
+
+Federation and content discovery still happens without relays, but they create an express-lane between instances who participate.
```
Comment (djwfyi) [inline docus/content/infrastructure/relays.md:16]:
```suggestion
Federation and content discovery still happen without relays, but relays create an express lane between instances who participate.
```

The `they` is ambiguous. And `happen` needs to be plural form.

File: docus/content/infrastructure/relays.md [hunk 2]
```diff
@@ -12,3 +12,28 @@ vmst.io does not currently run a Fediverse relay, but we participate in the foll
...
+
+## Relayed Instances
+
+Through participation in relays, these instances (and many more) are accessible in the Federated timeline for vmst.io users.
+_This list is subject to change at anytime due to circumstances outside of vmst.io administrators control._
```
Comment (djwfyi) [inline docus/content/infrastructure/relays.md:21]:
```suggestion
_This list is representative, incomplete, and subject to change at any time due to circumstances outside of vmst.io administrator control._
```

**Review by djwfyi — APPROVED** [review]
Two suggestions on the relays page, otherwise it looks good.
Giving my approval stamp a whirl.

**Pull request merged**